src/components/Articles/Articles.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import Article from '../Article/Article';
+import styles from './articles.module.scss'
+import thumbnailImage from '../../demo.png';
+
+class Articles extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      articles: []
+    }
+  }
+
+  componentWillMount() {
+    this.setState({
+      articles: [
+        {
+          id: 1,
+          title: 'This Article talks about some Tall Buildings And City Night Life',
+          slug: 'this-is-slug',
+          body: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
```

Comment:
Is there any reason why you are setting the state in componentWillMount? I will suggest you set this directly in the initial state i.e `this.state = { articles: [ ......... ] }`